### ISSUE TITLE
[TW-490] create new task with reverse dependency

### ISSUE THREAD

**Issue by tbabej** [tbabej/testimportfinal#462]
_Uli Martens on 2011-04-26T17:21:40Z says:_ 

 Often, I have a task in my task list and need to create a new task which has a dependency on it from the existing task. Right now I have to do this:

<pre>
$ task list
ID Project Pri Due Active Age    Description
 1                        4 secs bake a cake
$ task add buy milk
Created task 2.
$ : look for the new task ID in the above output, needed below
$ task dep:2 1
Modified 1 task
$task unblocked
ID Deps Project Pri Due Active Age   Description
 2                             1 min buy milk
$ 
</pre>

It would be much easier to have a new virtual argument "revdep" like this:

<pre>
$ task
ID Project Pri Due Active Age    Description
 1                        3 mins bake a cake
$ task add buy milk revdep:1
Created task 2.
Modified task 1.
$ task unblocked
ID Deps Project Pri Due Active Age   Description
 2                             1 min buy milk
$ 
</pre>
